conda-store-server/tests/_internal/worker/test_build.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) conda-store development team. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+from conda_store_server import api
+from conda_store_server._internal import schema
+from conda_store_server._internal.worker import build
+
+
+def test_build_started(db, seed_conda_store):
+    test_build = api.get_build(db, build_id=4)
```

Comment:
Might be good to check that it _isn't_ started before you do the thing that is supposed to start it.

Reply:
good idea! pushed up a change for this